recommender/services/deserializer.py

```diff
@@ -78,8 +80,12 @@ def deserialize_user_action(cls, json_dict):
         action_data = json_dict.get("action", {})
         action = Action(**action_data)
 
+        user_id, aai_uid = json_dict.get("user_id"), json_dict.get("aai_uid")
+        if aai_uid and not user_id:
```

Comment:
Assumption that if aai_token is passed, user_id is not. Just thinking loudly. No action is required.
So in the MP we cannot pass also aai_token